step-strikers-app/step-strikers-app/Classes/Armor.swift

```diff
@@ -0,0 +1,90 @@
+//
+//  Armor.swift
+//  step-strikers-app
+//
+//  Created by Jalyn Derry on 2/9/23.
+//
+
+
+protocol Armor {
+    var name: String { get}
+    var armorClass: Int {get}
+    
+    func checkIfSuited(potentialWearer: RPGCharacter) -> Bool
+}
+
+// This function will return the modified armor class in case the wearer is ill-suited for their currArmor
+func modifyArmorClass(wearer: RPGCharacter) -> Int {
```

Comment:
Better name might be calculateArmorClass()? Not a big deal but I saw you use this later in your code and I thought you were actually trying to modify something when actually you're just calculating and returning an int